docs/reference/setup/install/windows.asciidoc

```diff
@@ -467,13 +468,13 @@ start /wait msiexec.exe /i elasticsearch-{version}.msi /qn /l upgrade.log
 The `.msi` package handles uninstallation of all directories and files added as part of installation.
 
 WARNING: Uninstallation will remove **all** directories and their contents created as part of
-installation, **including data within the data directory**. If you wish to retain your data upon
-uninstallation, it is recommended that you make a copy of the data directory beforehand or consider using the snapshot API.
+installation, **except for data, config or logs directories**. If you wish to retain your data upon
+uninstallation, it is recommended that you make a copy of your data directory beforehand or consider using the snapshot API.
```

Comment:
> If you wish to retain your data upon uninstallation, it is recommended that you make a copy of your data directory

This doesn't align with the previous sentence, since data directory is not removed on uninstall

Reply:
Yes. I think asking the user to backup their data on an upgrade is probably good advice anyway, so will rephrase.